### ISSUE TITLE
reduce rules for verbpdgm3 and verbpdgm4

### ISSUE THREAD

**Issue by balmas** [alpheios-project/inflection-tables#336]
...
for the verbpdgm3 and 4, I need sometimes to review it, I strongly believe there should NOT be a verbpdgm3,1,verb,aw_fut,,,,,,
verbpdgm4,2,verb,aw_fut,,,,,contr,

I would do like this
verbpdgm3     1       verb reg_fut
verbpdgm4     2      verb ew_fut 
verbpdgm5	3	verb	aw_fut						

flag contr is not used anyway.

_Originally posted by @monzug in https://github.com/alpheios-project/inflection-tables/issues/289#issuecomment-560420110_

**Comment by balmas** [comment]
I'm still not convinced we shouldn't add in the morphflags here.

**Comment by irina060981** [comment]
@balmas and @monzug , I have made this changes - 
i have put only this 
```
verbpdgm3,1,verb,reg_fut,,,,,,
verbpdgm4,2,verb,ew_fut,,,,,,
verbpdgm5,3,verb,aw_fut,,,,,,
```

And made some lookups - look like it works well

![image](https://user-images.githubusercontent.com/12377640/72894143-6180a200-3d66-11ea-8c65-dee2315fe1c4.png)

![image](https://user-images.githubusercontent.com/12377640/72894224-8c6af600-3d66-11ea-8036-55f0750f269b.png)

(but here we have not all correct match)

![image](https://user-images.githubusercontent.com/12377640/72894269-a573a700-3d66-11ea-94e2-2bd6d4447ac4.png)

![image](https://user-images.githubusercontent.com/12377640/72894297-b6241d00-3d66-11ea-941d-d6d4bf63f2a0.png)

![image](https://user-images.githubusercontent.com/12377640/72894322-c2a87580-3d66-11ea-99ba-9a6bdaf0b0b1.png)

![image](https://user-images.githubusercontent.com/12377640/72894344-ce943780-3d66-11ea-8e26-a0eb3d764482.png)

![image](https://user-images.githubusercontent.com/12377640/72894359-d94ecc80-3d66-11ea-81a0-59cdd555351a.png)

![image](https://user-images.githubusercontent.com/12377640/72894376-e4096180-3d66-11ea-8050-494c24734e1b.png)

![image](https://user-images.githubusercontent.com/12377640/72894395-f1bee700-3d66-11ea-861b-eada6513d2b0.png)

![image](https://user-images.githubusercontent.com/12377640/72894416-00a59980-3d67-11ea-95c1-0a3ac8a0b329.png)

May be this information would be helpful.

**Comment by irina060981** [comment]
@balmas , @monzug 
I have found such a rule 

``
verbpdgm3,3,verb,aor_pass,passive,,future,,,
``

between `verbpdgm10` and `verbpdgm11`

May be it mistake?

**Comment by monzug** [comment]
no, it's not. it's for the future passive such as βουλευθήσομαι   and it works. will check your screenshots.

**Comment by monzug** [comment]
it looks like it works well except for the reg_fut. as u can see in the screenshots the active participle and middle participle are also highlighted and they should NOT. there is a rules.csv for the verb_participle too but the exampls in the screenshots are NOT verb_participle.  any idea what else has changed?
also, I don't know if the match order should be 1,2,3 or just 1 for all the 3 tables such as:
verbpdgm3,1,verb,reg_fut,,,,,,
verbpdgm4,1,verb,ew_fut,,,,,,
verbpdgm5,1,verb,aw_fut,,,,,,

**Comment by irina060981** [comment]
>as u can see in the screenshots the active participle and middle participle are also highlighted and they should NOT. there is a rules.csv for the verb_participle too but the exampls in the screenshots are NOT verb_participle.

@monzug, according to the following definition for the `active participle`
```
{
                    "role": "data",
                    "tense": "future",
                    "voice": "active",
                    "value": "βουλεύσων, βουλεύσουσᾰ, βουλεῦσον",
                    "reflink": {
                        "text": "(see declension)",
                        "id": "verbpdgm54"
                    }
                }
```

it would be highlighted for
`future, active`
and that's all
we have inflection with future and active. 
We don't have `"part of speech": "verb_participle",` condition here.

**Comment by monzug** [comment]
right now, if I lookup for    βουλεύσω  in the release build, it shows the following (your first screenshot):
![Screen Shot 2020-01-23 at 12 47 24 PM](https://user-images.githubusercontent.com/41396793/72982073-8d3f6e80-3dde-11ea-9e57-a60ade58fbfc.png)
the table is wrong and this issue will fix that, but you can see that the active participle is not highlighted. so, why is that in your environment the active principle for βουλεύσω is highlighted?

**Comment by monzug** [comment]
while testing the matching from the most common Greek words from the DCC list, I come across an other good example that we need to fix rules.csv for the future tables:
lookup γενήσομαι  it will bring up the Future System (Active and Middle) with contraction in -έω
but it should open the reg_fut table, Future System (without contraction) table as the stem_type is reg_fut
